internal/querycoordv2/observers/collection_observer.go

```diff
@@ -45,12 +47,16 @@ func NewCollectionObserver(
 	dist *meta.DistributionManager,
 	meta *meta.Meta,
 	targetMgr *meta.TargetManager,
+	collectionLoadedMap map[int64]int,
+	partitionLoadedMap map[int64]int,
```

Comment:
these new fields shouldn't be added to the constructor parameters, they are internal states, just initialize them at line 58

Reply:
ok